### PR TITLE
Add toISOString as alias for toISO

### LINKED ISSUE #20: toISOString as alias for toISO
`toISOString` is what's used by the native `Date` type and [`moment.duration`](https://momentjs.com/docs/#/durations/as-iso-string/). For consistency with those, I think adding `toISOString` as an alias of `toISO` makes sense. I know I find myself typing `toISOString()` on PostgresInterval objects a lot. 

Of course, then `toPostgresString` would have to be added for consistency with `toISOString` :)

### PR DESCRIPTION
**Pull request by gabegorelick**
Closes #20

### CODE REVIEW

**Comment by bendrucker** [comment]
Can you update the docs to reflect this? I'd actually rather consider `toISO` an alias of `toISOString`. I'd be 👍 for renaming if not for backwards compatibility.

**Comment by gabegorelick** [comment]
@bendrucker Docs updated. Let me know if that's what you had in mind.

File: test.js [hunk 1]
```diff
@@ -44,5 +44,6 @@ test(function (t) {
   t.equal(interval('00:00:00.100500').toISO(), 'P0Y0M0DT0H0M0.1005S')
   t.equal(interval('00:00:00.123456').toISO(), 'P0Y0M0DT0H0M0.123456S')
   t.equal(interval('-00:00:00.123456').toISO(), 'P0Y0M0DT0H0M-0.123456S')
+  t.equal(interval('-00:00:00.123456').toISOString(), 'P0Y0M0DT0H0M-0.123456S')
```
Comment (bendrucker) [inline test.js:46]: Can you update the tests as well to flip toISO/toISOString? Otherwise LGTM.

Comment (gabegorelick) [reply test.js:46]: Done.

**Pull request merged**